src/pages/api/user/updateUser.ts

```diff
@@ -0,0 +1,43 @@
+import { PrismaClient } from '@prisma/client';
+
+export default async function handler(req, res) {
+    const prisma = new PrismaClient();
+
+    // Check the HTTP method and if we have a body
+    if (req.method !== 'POST') {
+        return res.status(405).json({message: 'HTTP method not valid - only POST Accepted'});
+      }
+    if (!req.body) {
+    return res.status(404).json({error: "Don't have form data."});
+    }
+
+    // Get the data from the body (just name and type for now)
+    const {email, name, type} = req.body;
+
+    if (!type || type === '' || !email) {
+        return res.status(422).json({message: 'User type or email is missing'});
```

Comment:
What if they're only updating one field? Might be preferable to do PATCH instead of POST

Reply:
[eventId] has an example if you change it

Reply:
We both kinda wrote the same api, so we'll need to decide which way to go, but we can punt a little longer, or just keep them both and have them be redundant

Reply:
I'm gonna punt on this one too for now